src/zeusops_bot/cli.py

```diff
@@ -43,10 +47,10 @@ def cli(arguments: list[str] | None = None):
     if arguments is None:
         arguments = sys.argv[1:]
     _args = parse_arguments(arguments)
-    return main()
+    return main(_args)
 
 
-def main():
+def main(args: argparse.Namespace):
```

Comment:
I'd reject the main func needing CLI-lib objects, instead pass any objects needed directly.

```suggestion
def main(debug: bool=False):
```

or so

Reply:
Yeah, your reasoning makes sense. I was considering between these two options and happened to pick the other one, will edit.

Reply:
This has been fixed now